Mathlib/Topology/Algebra/Algebra/Equiv.lean

```diff
@@ -302,6 +302,25 @@ theorem _root_.AlgEquiv.isUniformEmbedding {E₁ E₂ : Type*} [UniformSpace E
     continuous_toFun := h₁
     continuous_invFun := by dsimp; fun_prop }
 
+/-- Interpret an algebra equivalence as a continuous algebra equivalence when it is continuous. -/
+def ofAlgEquiv (e : A ≃ₐ[R] B) (he : Continuous e) (he' : Continuous e.symm) : A ≃A[R] B :=
+  { e with continuous_toFun := he, continuous_invFun := he' }
```

Comment:
This is just the pre-existing constructor for `ContinuousAlgEquiv`:
```
ContinuousAlgEquiv.mk.{u_1, u_2, u_3} {R : Type u_1} {A : Type u_2} {B : Type u_3} [CommSemiring R] [Semiring A]
  [TopologicalSpace A] [Semiring B] [TopologicalSpace B] [Algebra R A] [Algebra R B] (toAlgEquiv : A ≃ₐ[R] B)
  (continuous_toFun : Continuous toAlgEquiv.toFun := by fun_prop)
  (continuous_invFun : Continuous toAlgEquiv.invFun := by fun_prop) : A ≃A[R]
```
(up to some defeqs involving `toAlgEquiv.invFun`, `AlgEquiv.symm` and `DFunLike.coe`). So I think you can remove all this and use `mk` instead.